docs/installation_linux.rst

```diff
@@ -29,14 +29,12 @@ To check which version of Python is used on your system, run the following:
 If you need to install a different Python package, there are likely packaged versions for
 your distro.
 
-Here is an example to install Python 3.10 on Ubuntu 20.04
+Here is an example to install Python 3.11 on Ubuntu 22.04
 
 .. code:: bash
 
    sudo apt update
-   sudo apt install software-properties-common -y
-   sudo add-apt-repository ppa:deadsnakes/ppa -y
-   sudo apt install python3.10 python3.10-dev -y
+   sudo apt install python3.11 python3.11-dev -y
```

Comment:
What's the reason to add -dev?

Reply:
I was being consistent with the existing example, but looking at the package, the -dev package seems to mainly have additional documentation. We can probably drop it.